packages/kv-store/src/Redis/Redis.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Tempest\KeyValue\Redis;
+
+use Stringable;
+use Tempest\DateTime\DateTimeInterface;
+use Tempest\DateTime\Duration;
+
+interface Redis
```

Comment:
Do we need an interface? If so, I feel like `Redis` is not an accurate enough name. Is this a client that interacts with a Redis server? Is this a gateway that knows how to talk to it? IDK

Reply:
Edit, having seen the implementations, I think `RedisClient` is more accurate

Reply:
I initially chose `Redis` as a name because I how DI looks:

```php
public function __construct(
    public readonly Redis $redis,
) {}
```

> Is this a client that interacts with a Redis server? Is this a gateway that knows how to talk to it?

Both sound the same to me. Can we keep `Redis`? 😁

Reply:
Come to think of it, we also call the interface that connects with the database simply `Database`, so I reckon this is fine :)